.gitignore

```diff
@@ -1,343 +1,345 @@
-## Ignore Visual Studio temporary files, build results, and
```

Comment:
what's going on here? is there an actual change?

Reply:
oh i dont know why like this 
but yes - added last row
**/quick_start_browser/package-lock.json

Reply:
probably line ending again